system/loggerd/loggerd.h

```diff
@@ -93,6 +74,7 @@ const EncoderInfo qcam_encoder_info = {
   .frame_height = 330,
   .get_encode_data_func = &cereal::Event::Reader::getQRoadEncodeData,
   .set_encode_idx_func = &cereal::Event::Builder::setQRoadEncodeIdx,
+  .init_encode_data_func = &cereal::Event::Builder::initQRoadEncodeData,
```

Comment:
might be nice to setup a macro for this so you'd have to specify "QRoadEncod" once